### PR TITLE
feat: forester alerts + monitoring

### PR DESCRIPTION
**Pull request by sergeytimoshin**
Added:

• Tracing-based telemetry
• Prometheus metrics sender
• Prometheus + AlertManager + PushGateway + Grafana docker-compose configuration

### CODE REVIEW

File: forester/docker-compose.yml [hunk 1]
```diff
@@ -22,7 +20,71 @@ services:
       - PGADMIN_DEFAULT_PASSWORD=photon
```
Comment (ananas-block) [inline forester/docker-compose.yml:20]: should we make this an env variable?
(Same question for postgres login above.)

File: forester/docker-compose.yml [hunk 2]
```diff
@@ -22,7 +20,71 @@ services:
...
+      - ./alert_rules.yml:/etc/prometheus/alert_rules.yml
+    networks:
+      - forester-net
+
+  grafana:
+    image: grafana/grafana
+    ports:
+      - "3000:3000"
+    environment:
+      - GF_SECURITY_ADMIN_PASSWORD=admin
```
Comment (ananas-block) [inline forester/docker-compose.yml:78]: env variable necessary?

File: forester/src/epoch_manager.rs [hunk 3]
```diff
@@ -541,11 +527,24 @@ impl<R: RpcConnection, I: Indexer<R>> EpochManager<R, I> {
                     epoch_pda.epoch,
                 )
                 .await?;
+
+                // Prometheus metrics
+                let chunk_duration = start_time.elapsed();
+                queue_metric_update(epoch_info.epoch, num_tx_sent, chunk_duration).await;
+
+                // TODO: consider do we really need WorkReport
+                self.increment_processed_items_count(epoch_info.epoch, num_tx_sent)
```
Comment (ananas-block) [inline forester/src/epoch_manager.rs:536]: I see that we could use it to push things to graphana but in once we have multiple foresters by different parties we should get the performed work from the `ForesterEpochPda`s and the total performed work from the `EpochPda` .

File: test-utils/src/rpc/solana_rpc.rs [hunk 4]
```diff
@@ -345,6 +361,7 @@ impl RpcConnection for SolanaRpcConnection {
     }
 
     async fn get_slot(&mut self) -> Result<u64, RpcError> {
+        println!("Calling get_slot");
```
Comment (ananas-block) [inline test-utils/src/rpc/solana_rpc.rs:364]: shouldn't we call `info!` here?

File: forester/docker-compose.yml (hunk 1 above)
Comment (sergeytimoshin) [reply forester/docker-compose.yml:20]: yes we can, and probably should, it's just a default test value, it implies that the end user should replace it with something more secure

**Review by ananas-block — APPROVED** [review]

**Pull request merged**